examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -562,7 +562,9 @@ def transform_images(examples):
 
             clean_images = batch["input"]
             # Sample noise that we'll add to the images
-            noise = torch.randn(clean_images.shape).to(clean_images.device)
+            noise = torch.randn(
+                clean_images.shape, dtype=(torch.float32 if args.mixed_precision == "no" else torch.float16)
+            ).to(clean_images.device)
```

Comment:
Can you break these out as two different lines pls

Reply:
@williamberman done!